src/SqlPersistence/Outbox/TransactionScopeSqlOutboxTransaction.cs

```diff
@@ -33,7 +36,8 @@ public void Prepare(ContextBag context)
     {
         var options = new TransactionOptions
         {
-            IsolationLevel = isolationLevel
+            IsolationLevel = isolationLevel,
+            Timeout = transactionTimeout
```

Comment:
As `Timeout` is now always set it will now be set with `TimeSpan.Zero`, I looked in to the .framework source code and that is also the default
```suggestion
            Timeout = transactionTimeout; // TimeSpan.Zero is default of `TransactionOptions.Timeout`
```

Reply:
Yes. TransactionOptions is a struct. Previous code doesn't set Timeout, so it equals to default(TimeSpan) = TimeSpan.Zero